aws_marketplace/using_data/using_data_with_ml_model/using-dataset-product-from-aws-data-exchange-with-ml-model-from-aws-marketplace.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Use <code>you</code> instead of <code>we</code>/<code>us</code> throughout the notebook. The customer is doing the implementation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>